projects/ngneat/transloco-locale/src/lib/transloco-locale.types.ts

```diff
@@ -0,0 +1,47 @@
+export interface NumberFormatOptions {
+  /**
+   * Whether to use grouping separators, such as thousands separators or thousand/lakh/crore separators. Possible values are true and false; the default is true.
+   */
+  useGrouping?: Intl.NumberFormatOptions['useGrouping'];
+  /**
+   * The minimum number of integer digits to use. Possible values are from 1 to 21.
+   */
+  minimumIntegerDigits?: Intl.NumberFormatOptions['minimumIntegerDigits'];
+  /**
+   * The minimum number of fraction digits to use. Possible values are from 0 to 20.
+   */
+  minimumFractionDigits?: Intl.NumberFormatOptions['minimumFractionDigits'];
+  /**
+   * The maximum number of fraction digits to use. Possible values are from 0 to 20.
+   */
+  maximumFractionDigits?: Intl.NumberFormatOptions['maximumFractionDigits'];
+  /**
+   * The minimum number of significant digits to use. Possible values are from 1 to 21.
+   */
+  minimumSignificantDigits?: Intl.NumberFormatOptions['minimumSignificantDigits'];
+  /**
+   * The maximum number of significant digits to use. Possible values are from 1 to 21
+   */
+  maximumSignificantDigits?: number;
+}
+

```

Comment:
The `minimum` and `maximum` prefixes are long and error-prone. What do you think about using `min` and `max` instead?